pydarn/plotting/grid.py

```diff
@@ -82,12 +81,6 @@ def plot_grid(cls, dmap_data: List[dict], record: int = 0,
             parameter: str
                 Key name indicating which parameter to plot.
                 Default: vel (Velocity). Alternatives: 'pwr', 'wdt'
-            lowlat: int
-                Lower AACGM latitude boundary for the polar plot
-                Default: 50
-            fov: bool
-                Set to false to not plot the outline of the FOV
-                Default: True
```

Comment:
If I fix the dtime error I then get:
```
line 164, in plot_grid
    boundary=fov,
NameError: name 'fov' is not defined
```
I think maybe fov kw shouldn't be contained in kwargs as we want to be able to set a default if not given?